src/status_im/contexts/wallet/tokens/events.cljs

```diff
@@ -68,9 +78,10 @@
             :on-error   [:wallet.tokens/fetch-prices-failed]}]]
      :db (-> db
              (assoc-in [:wallet :tokens]
-                       {:sources    (:sources tokens)
-                        :by-address (-> tokens :by-address vals)
-                        :by-symbol  (-> tokens :by-symbol vals)})
+                       {:sources                    (:sources tokens)
+                        :supported-chains-by-symbol supported-chains-by-symbol
+                        :by-address                 (-> tokens :by-address vals)
+                        :by-symbol                  by-symbol-vals})
```

Comment:
Not a change you introduced, but what's the point of grouping the tokens by _symbol_ and by _address_ (as a `map` of symbols/tokens and token data), if we dismiss that and use the vector of token data (with the symbol/token as a key in each map)? I assume it was originally done to make extraction of token data more performant and easier, without having to go through the entire list of tokens. Here though, it seems like we are storing the tokens as a vector in the db three times, and it's not clear to me why. The token list is pretty large, so storing it 3 times seems excessive tbh.

Reply:
I guess @alwx is the best person to answer.

Reply:
The original idea was to copy this logic from desktop :) It was part of a bigger issue and I assumed that it's better to do all of it at once.